### DISCUSSION TITLE
Error when compiling - pdfLaTeX+MakeIndex+BibTeX

### DISCUSSION THREAD

**Discussion by FlorBotta** [MiKTeX/miktex#908] [General]
Hello,

I am new to Latex and after updating the software I could not get any code to work, not even a basic test one. I get these errors when I compile in pdfLaTeX+MakeIndex+BibTeX

> Sorry, but "MiKTeX Compiler Driver" did not succeed.
> 
> The log file hopefully contains the information to get MiKTeX going again:
> 
>   C:/Users/xxxxx/AppData/Local/MiKTeX/2.9/miktex/log/texify.log
> 
> You may want to visit the MiKTeX project page, if you need help.
> 

When I checked the log it said:

> 2021-08-26 00:48:30,001+0100 INFO  texify - starting with command line: "C:\Program Files\MiKTeX 2.9\miktex\bin\x64\texify.exe" --pdf --synctex=1 --clean untitled-1.tex
> 2021-08-26 00:48:30,758+0100 FATAL texify - TeX engine failed for some reason.
> 2021-08-26 00:48:30,758+0100 FATAL texify - Info: 
> 2021-08-26 00:48:30,758+0100 FATAL texify - Source: Programs\MiKTeX\texify\mcd.cpp
> 2021-08-26 00:48:30,903+0100 FATAL texify - Line: 1476
> 2021-08-26 00:53:25,785+0100 INFO  texify - starting with command line: "C:\Program Files\MiKTeX 2.9\miktex\bin\x64\texify.exe" --pdf --synctex=1 --clean untitled-1.tex
> 2021-08-26 00:53:26,024+0100 FATAL texify - TeX engine failed for some reason.
> 2021-08-26 00:53:26,024+0100 FATAL texify - Info: 
> 2021-08-26 00:53:26,024+0100 FATAL texify - Source: Programs\MiKTeX\texify\mcd.cpp
> 2021-08-26 00:53:26,024+0100 FATAL texify - Line: 1476
> 

I don't know how to interpret this error nor how to fix it.

Any advice is welcomed.

Thank you,
Sincerely,

**Comment by BenelM** [comment]
Greetings. Have you sorted out the issue? I am facing the same problem after re-installing MiKTeX. Thanks.
